library/nbde_server_tang.py

```diff
@@ -74,7 +147,7 @@ def create_keys(module, keygen, keydir, force):
             result = dict(msg="Listing keys failed: {}".format(to_native(exc)))
             raise TangAnsibleError(result)
 
-    result = {"changed": True, "state": "keys-created"}
+    result = {"changed": True}
     if not module.check_mode:
         result = generate_tang_keys(module, keygen, keydir)
     return result
```

Comment:
so the intention here is that in `check_mode`, the module will always report `changed: True`?

Reply:
Yes, to indicate that a change would have been performed, if it was not running under check mode.

Reply:
To clarify, this method could have returned earlier, at which point it would report changed False. At this point, a change will be made, so I only do the change if not running under check mode, but still report a change regardless.